compiler/src/dmd/cparse.d

```diff
@@ -69,6 +69,13 @@ final class CParser(AST) : Parser!AST
      */
     OutBuffer* defines;
 
+    // Identifier and source location pair used by cparseDeclarator
+    private static struct IdentifierWithLoc
```

Comment:
One part of me something like this must exist somewhere else, but can't see one in any other module.